doc/telescope.txt

```diff
@@ -269,20 +269,530 @@ actions.open_qflist()                                  *actions.open_qflist()*
 ================================================================================
                                                              *telescope.builtin*
 
-A collection of builtin pickers for telescope.
-
-Meant for both example and for easy startup.
+Telescope Builtins is a collection of community maintained pickers to support
+common workflows. It can be used as reference when writing PRs, Telescope
+extensions, your own custom pickers, or just as a discovery tool for all of the
+amazing pickers already shipped with Telescope!
 
 Any of these functions can just be called directly by doing:
 
-:lua require('telescope.builtin').$NAME()
+:lua require('telescope.builtin').$NAME_OF_PICKER()
+
+To use any of Telescope's default options or any picker-specific options, call
+your desired picker by passing a lua table to the picker with all of the
+options you want to use. Here's an example with the live_grep picker:
+
+:lua require('telescope.builtin').live_grep({
+   prompt_title = 'find string in open buffers...',
+   grep_open_files = true
+ })
```

Comment:
😍

Reply:
I think we should take this space to talk about what all picker share in common. Like setting custom mappings, picker title ... etc. Not sure if it should be part of this pr as you've already did ALLLOT of amazing work

Reply:
I agree we should do that. I think that would probably be a good thing to do in my next PR, which will be focused on cleaning up the README a bit more. Just so we can get this merged ASAP! But I will **definitely** do that, I think this is a great place to put that info

Reply:
Awesome @smithbm2316 , I'm sooo glad that this is being done. There are a lot of goodies in telescope that I was anxious that will be lost with time. Thank you